.env.template

```diff
@@ -4,3 +4,4 @@ MLFLOW_TRACKING_URI="http://127.0.0.1:5000"
 AWS_ACCESS_KEY_ID=your_access_key_id
 AWS_SECRET_ACCESS_KEY=your_secret_access_key
 AWS_ENDPOINT=your_endpoint_url
+AWS_S3_BUCKET=your_bucket_name
```

Comment:
Could you clarify why we need both AWS_ENDPOINT and AWS_S3_BUCKET? Is there any use case where these might refer to different buckets? Or if they always refer to the same bucket, then why can't we always connect using the endpoint URL (which contains the bucket name, if I recall correctly)?

Reply:
Geowerkstatt is using this structure for its environment variables, so I need to comply with it. When we developed the code, we passed all the information into the `AWS_ENDPOINT` variable, but Oliver told me that they prefer to do it this way. I think that this indeed leaves you the flexibility to open several S3 buckets from the same AWS account by adapting the bucket name. 

If you have a strong opinion about having everything in the `AWS_ENDPOINT` then I can ask Oliver whether it is possible for them to adapt their code.